client/src/components/ErrorPage.tsx

```diff
@@ -0,0 +1,60 @@
+import {
```

Comment:
this would be better in one line I think

Reply:
No